src/modules/cl/cl_geometry_transforms.cpp

```diff
@@ -437,17 +430,127 @@ resize_crop_cl_batch(cl_mem srcPtr, cl_mem dstPtr, rpp::Handle &handle,
                                                                         handle.GetInitHandle()->mem.mgpu.dstInc,
                                                                         padding,
                                                                         type,
-                                                                        plnpkdind);
+                                                                        plnpkdind, plnpkdind);
+    return RPP_SUCCESS;
+}
+
+RppStatus
+rotate_cl_batch_tensor(cl_mem srcPtr, cl_mem dstPtr, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
+{
+    int in_plnpkdind = getplnpkdind(tensor_info._in_format), out_plnpkdind = getplnpkdind(tensor_info._out_format);
+    int batch_size = handle.GetBatchSize();
```

Comment:
Codacy found an issue: [Variable 'batch_size' is assigned a value that is never used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/rpp/pullRequest?prid=5976402)